src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+  }
```

Comment:
Specify for which properties you're applying transition, move time to the var(you used it 2 times)